homeassistant/components/recorder/__init__.py

```diff
@@ -34,6 +35,10 @@
 
 CONF_DB_URL = 'db_url'
 CONF_PURGE_DAYS = 'purge_days'
+CONF_EXCLUDE = 'exclude'
+CONF_INCLUDE = 'include'
+CONF_ENTITIES = 'entities'
+CONF_DOMAINS = 'domains'
```

Comment:
Can you import these from history? Or better yet if only in history move it to consts and import from there?